ihs/src/main/scripts/install.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/sh
+
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#           http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
```

Comment:
Why note have the set and IFS here?

Reply:
Good question. This script intends to be executed by `sh` instead of `bash`, indicated by the 1st line `#!/bin/sh`.